src/components/CommentList/CommentList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Comment from '../Comment/Comment';
+import './CommentList.css';
+
+function CommentList({ comments }) {
+  const commentList = comments.map(
+    comment => <Comment key={comment.id} comment={comment} />
+  );
+
+  return <div className="comment-list">{commentList}</div>;
```

Comment:
it's not necessary to create separate variable for this

Reply:
fixed